taworks/taform/models.py

```diff
@@ -105,9 +105,15 @@ class Meta:
 class ApplicationForm(ModelForm):
     class Meta:
         model = Application
-        exclude = ('student', 'course', 'application_date', 'instructor_preference', )
-        widgets = {'reason': Textarea(attrs={'cols':50, 'rows':1}), 'preference': Textarea(
-            attrs={'cols':15,' rows':1})}
+        exclude = ('student', 'course', 'application_date', )
```

Comment:
I think we still want to exclude instructor preference right? It's showing up the form right now but isnt needed here.
![screen shot 2018-02-04 at 6 18 03 pm](https://user-images.githubusercontent.com/22120696/35783601-ff41d3ac-09d7-11e8-8ad4-2da3eefc82ca.png)

Reply:
definitely. i messed up in my merge.